templates/home/home-en.html

```diff
@@ -167,21 +143,74 @@ <h2>Your government</h2>
 		</div>
 	</div>
 </section>
-<section class="container gc-features">
-	<h2>Government initiatives</h2>
-	<div class="row wb-eqht">
-		<div class="col-sm-6">
-			<div class="well well-sm brdr-rds-0 eqht-trgt">
-				<img class="img-responsive full-width" src="../../components/gc-features/img/initiative-520x200.png" alt=""/>
-				<h3 class="h5"><a class="stretched-link" href="#">[Feature hyperlink text]</a></h3>
-				<p>Brief description of the feature being promoted.</p>
+
+<section class="gc-features">
+	<div class="container">
+		<h2>Government initiatives</h2>
+		<!-- START: Add the list of links to the features to display in basic HTML mode -->
+		<ul class="nojs-show">
+			<li><a href="#">[1. Feature hyperlink text]</a></li>
+			<li><a href="#">[2. Feature hyperlink text]</a></li>
+			<li><a href="#">[3. Feature hyperlink text]</a></li>
+			<li><a href="#">[4. Feature hyperlink text]</a></li>
+			<li><a href="#">[5. Feature hyperlink text]</a></li>
+			<li><a href="#">[6. Feature hyperlink text]</a></li>
+			<li><a href="#">[7. Feature hyperlink text]</a></li>
+			<li><a href="#">[8. Feature hyperlink text]</a></li>
+		</ul>
+		<!-- END -->
+
+		<!-- Add the "data-wb-randomize" attribute to the "col-sm-6" element to randomize the features within each theme -->
+		<div class="row wb-eqht-grd">
+			<div class="col-sm-6" data-wb-randomize='{ "toggle": "hide" }'>
+				<div class="well well-sm eqht-trgt hide">
+					<img src="../assets/home-feature-1-520x200.jpg" alt="">
+					<h3><a class="stretched-link" href="#">[1. Feature hyperlink text]</a></h3>
+					<p>1. Brief description of the feature being promoted.</p>
+				</div>
+				<div class="well well-sm eqht-trgt hide">
+					<img src="../assets/home-feature-2-520x200.jpg" alt="">
+					<h3><a class="stretched-link" href="#">[2. Feature hyperlink text]</a></h3>
+					<p>2. Brief description of the feature being promoted.</p>
+				</div>
+			</div>
+			<div class="col-sm-6" data-wb-randomize='{ "toggle": "hide" }'>
+				<div class="well well-sm eqht-trgt hide">
+					<img src="../assets/home-feature-3-520x200.jpg" alt="">
+					<h3><a class="stretched-link" href="#">[3. Feature hyperlink text]</a></h3>
+					<p>3. Brief description of the feature being promoted.</p>
+				</div>
```

Comment:
I do think we will need an explicit section here in order to associate the decorative image with its proper heading. A div have no semantic.

Like this, the image belong to the preceding title

```suggestion
				<section class="well well-sm eqht-trgt hide">
					<img src="../assets/home-feature-3-520x200.jpg" alt="">
					<h3><a class="stretched-link" href="#">[3. Feature hyperlink text]</a></h3>
					<p>3. Brief description of the feature being promoted.</p>
				</section>
```